web/styles/subscriptions.css

```diff
@@ -1435,32 +1437,30 @@ div.settings-radio-input-parent {
         .two-pane-settings-header .fa-chevron-left {
             visibility: visible;
         }
-    }
-
-    .two-pane-settings-overlay .left,
-    .two-pane-settings-overlay .right {
-        position: absolute;
-        display: block;
-        margin: 0;
-        width: 100%;
-        height: calc(100% - var(--settings-overlay-header-height));
 
-        border: none;
-    }
-
-    .two-pane-settings-overlay .right {
-        position: absolute;
-        left: 101%;
+        /* default left pane open without `.right-pane-open` present */
+        .left {
+            display: block;
```

Comment:
I'd really like us to get away from the display block/none method of showing and hiding things. It might be okay in this instance, but we just can't know for sure that an element isn't `display: grid` or `display: flex`--and a careless `display: block` can wreak havoc on such layouts.